crates/runtime/src/runtime/runtime.rs

```diff
@@ -707,6 +707,34 @@ impl Runtime {
         let template_addr = account_storage.template_addr().unwrap();
         Some((balance, counter, template_addr))
     }
+
+    /// Sends coins from the current executing account to a destination account.
+    ///
+    /// # Panics
+    ///
+    /// Panics when the destination account does not exist.
+    pub fn transfer(&self, src_addr: &Address, dst_addr: &Address, amount: u64) {
+        let mut src_account = AccountStorage::load(self.gs.clone(), src_addr).unwrap();
+
+        let mut dst_account = if let Some((_bal, _counter, _addr)) = self.get_account(dst_addr) {
+            AccountStorage::load(self.gs.clone(), dst_addr).unwrap()
+        } else {
+            panic!("Destination account does not exist")
```

Comment:
@neysofu this soon will have to be modified to create a `Stub Account` as detailed in the [`Self-Spawn SMIP`](https://github.com/spacemeshos/SMIPS/issues/64)

Reply:
Yup, will have to open a PR soon! For now we're just panicking.

Reply:
https://github.com/spacemeshos/svm/issues/424.